### PR TITLE
Features/consumer checkout screens

### PR DESCRIPTION
**Pull request by manhenc**
Thank you for your contribution to the repo. 
Before submitting this PR, please make sure:

- [x] Your code builds clean without any errors or warnings
- [x] You are using approved terminology
- [x] You have added unit tests

You may delete the sections as necessary.

### Context
In this PR, the 3 consumer checkout screens(address, shipping, and payment) is being added. The first screen, address screen, will be routed from the checkout button in the consumer order screen. In the address screen, the address card is implemented in a ListView.builder which will work along with the "delete address" button in the card. The "delete address" button will return a alertdialog when pressed and will remove the selected address card if "Remove" button is pressed. 

Note: 
The "Pay" button in payment screen is grey as onPressed() is set to null.
Regular checkbox is being used as a better alternative for circular check box has not been found yet. 

image
image
image
image

### Changes
The "checkout" button in consumer order screen is added with routing to consumer checkout address screen. 
More texts are added into the localization files.

### Shoutouts
Joseph and JiaYuan

### CODE REVIEW

File: lib/screens/consumer_checkout/consumer_checkout_address_screen.dart [hunk 1]
```diff
@@ -0,0 +1,363 @@
...
+      state: VueStepState.disabled,
+    ),
+    VueStep(
+      title: Text(LocaleText.textPayment.tr()),
+      content: Container(),
+      state: VueStepState.disabled,
+    ),
+  ];
+
+//  bool _addressCheck = false;
```
Comment (wheel1992) [inline lib/screens/consumer_checkout/consumer_checkout_address_screen.dart:43]: Remove unused comment

File: lib/screens/consumer_checkout/consumer_checkout_payment_screen.dart [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+          mainAxisAlignment: MainAxisAlignment.spaceAround,
+          children: <Widget>[
+            Offstage(
+              offstage: true,
+              child: Icon(Icons.lens, color: Colors.black, size: 15),
+            ),
+            Offstage(
+              offstage: true,
+              child: Icon(Icons.lens, color: Colors.black, size: 15),
+            ),
```
Comment (wheel1992) [inline lib/screens/consumer_checkout/consumer_checkout_payment_screen.dart:149]: Seems that similar codes are written in `consumer_checkout_address_screen.dart`. 

Consider putting in a new widget to handle design

File: lib/screens/consumer_checkout/consumer_checkout_payment_screen.dart [hunk 3]
```diff
@@ -0,0 +1,264 @@
...
+                      Padding(
+                        padding: const EdgeInsets.only(left: 4),
+                        child: Text('1458'),
+                      ),
+                    ],
+                  ),
+                ),
+                Checkbox(
+                    value: _cardCheck,
+                    activeColor: Colors.orangeAccent,
```
Comment (wheel1992) [inline lib/screens/consumer_checkout/consumer_checkout_payment_screen.dart:222]: Similar code structure, comparing to `_buildSelectCash()`.

Consider writing a more generic function.

File: lib/screens/consumer_checkout/consumer_checkout_shipping_screen.dart [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+        ),
+      );
+
+  Widget _buildShippingStepIndicatorCircle() => Positioned(
+        top: 48,
+        width: 430,
+        child: Row(
+          mainAxisAlignment: MainAxisAlignment.spaceAround,
+          children: <Widget>[
+            Offstage(
```
Comment (wheel1992) [inline lib/screens/consumer_checkout/consumer_checkout_shipping_screen.dart:150]: Same code structure here too

File: lib/util/constants.dart [hunk 5]
```diff
@@ -126,11 +128,15 @@ class LocaleText {
   static const textActivity = 'text.activity';
   static const textAdd = 'text.add';
   static const textAddPhoto = 'text.addPhoto';
+  static const textAddNewAddress = 'text.addNewAddress';
+  static const textAddNewPaymentMethod = 'text.addNewPaymentMethod';
```
Comment (wheel1992) [inline lib/util/constants.dart:132]: Haha, in alphabetic order

File: lib/screens/consumer_checkout/consumer_checkout_payment_screen.dart (hunk 2 above)
Comment (manhenc) [reply lib/screens/consumer_checkout/consumer_checkout_payment_screen.dart:149]: Created a widget produces a customisable step slider, this widget can take in any steps. 
It is customisable in terms of size, width, colour, and positioning of the slider and circles.

File: lib/screens/consumer_checkout/consumer_checkout_payment_screen.dart (hunk 3 above)
Comment (manhenc) [reply lib/screens/consumer_checkout/consumer_checkout_payment_screen.dart:206]: Written a generic widget that can be implemented with a ListView once there is a confirmed model for consumer payment details

**Pull request merged**